internal/daysteps/daysteps.go

```diff
@@ -1,13 +1,72 @@
 package daysteps
 
+import (
+	"fmt"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/Yandex-Practicum/tracker/internal/personaldata"
+	"github.com/Yandex-Practicum/tracker/internal/spentenergy"
+)
+
+// DaySteps содержит данные о дневной прогулке.
 type DaySteps struct {
-	// TODO: добавить поля
+	Steps                 int           // количество шагов
+	Duration              time.Duration // длительность прогулки
+	personaldata.Personal               // персональные данные пользователя
 }
 
+// Parse разбирает строку с данными о прогулке формата "678,0h50m" и заполняет поля структуры.
 func (ds *DaySteps) Parse(datastring string) (err error) {
-	// TODO: реализовать функцию
+	parts := strings.Split(datastring, ",")
+	if len(parts) != 2 {
+		return fmt.Errorf("некорректный формат данных")
+	}
+
+	// Парсим количество шагов
+	steps, err := strconv.Atoi(parts[0])
+	if err != nil {
+		return fmt.Errorf("некорректное значение шагов")
```

Comment:
1. Теряешь сообщение об ошибки
2. Ошибку лучше писать на английском
```suggestion
		return 0, 0, fmt.Errorf("conversion error: %w", err)
```